components/DynamicForm/MyResult.tsx

```diff
@@ -5,35 +5,51 @@ import {
   FormCtx,
   FormValues,
 } from '../../utils/FormContext'
-import { answers, AnswersKeys, questions, QuestionsKeys } from '../../models'
+import {
+  answers,
+  AnswersKeys,
+  Question,
+  /*questions,*/ QuestionsKeys,
+} from '../../models'
 import { QuestionText } from '../FormStyles/QuestionText'
 import styled from 'styled-components'
 import { StyledTextInput } from '../FormStyles/InputBox'
 
 interface MyResultProps {
   label: string
   description?: string
+  questions: Question[]
 }
 
 // translates ID-based results to content-based results
-export function buildResults(formAnswers: {
-  [key: string]: FormAnswer
-}): FormAnswer[] {
+export function buildResults(
```

Comment:
Notes for people reviewing: Ignore this part, these are changes from the other PR